### DISCUSSION TITLE
Should you use github as a place to share datasets?

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#543] [Q&A] [answered]
[Question from github basics training](https://github.com/fieldmuseum/EMu-Documentation/discussions/542) 

_Data on github / integrated in some ways over time, too? (JB/RB)_

**Comment by rondlg** [comment]
Yes/Possibly – useful tools for documenting decisions/background/might complement
Note - for taxonomic or collections data you need to publish (e.g. have assigned DOIs), GBIF (& other platforms like osf.io) may be more appropriate/necessary

**Comment by magpiedin** [comment, marked as answer]
@rbaquiran @fmrdelapena @jbates606 (a little follow-up):
GitHub.com can help handle versioning for working-draft-datasets, and can keep them in the context of analyses/code/discussions when needed, but two things:

1. For larger datasets -- consider storage limits on GitHub ([some current help-docs here](https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-storage-and-bandwidth-usage)).  

2. To publish a dataset to a more long-term-accessible home and assign a stable identifier (DOI, ARK, etc) and/or URL, sites like the following might be better suited:
    - [GBIF](gbif.org) to publish occurrence datasets or [taxonomic checklists](https://www.gbif.org/species/search?advanced=1) (via EMu & the [FMNH IPT](https://fmipt.fieldmuseum.org/ipt/) for in-house datasets) or 
    - [OSF](https://osf.io) (+/- [Dryad](https://datadryad.org), [Zenodo](https://zenodo.org), etc) to publish output from analyses or models that aren't in scope for aggregators/repo's like GBIF
